README.md

```diff
@@ -553,6 +703,188 @@ type: Opaque
+  password: ZmYwZDM4YWItN2IzMC00ODE1LWI5OTMtMjQwYTc3YjQwZmMw
+  bearer: ZXlKaGJHY2lPaUpJVXpJMU5pSXNJblI1Y0NJNklrcFhWQ0o5LmV5SjFjMlZ5SWpvaU5XSmlZemxrT0RFdFpqYzJNUzAwWVdFNUxXSmlNekl0WkRCaU9UVmtaR05rTXpkbUlpd2laWGh3SWpveE9UWXlOalUxT0RjNUxDSnBZWFFpT2pFMk1ESTJOVFU0Tnprc0ltbHpjeUk2SW10MVltVnpjR2hsY21VaUxDSnVZbVlpT2pFMk1ESTJOVFU0TnpsOS40Rk4wS3FIRF91Q1AtRmFIMmFpT3ZPUjFsY2wtVjFyS0Z4d2RQXzNuRmY0
+kind: Secret
+metadata:
+  labels:
+    app: notification-manager
+  name: default-webhook-secret
+  namespace: default
+type: Opaque
+
+---
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: WebhookConfig
+metadata:
+  name: default-webhook-config
+  namespace: default
+  labels:
+    app: notification-manager
+    type: default
+spec:
+  url: http://127.0.0.1:8080/
+  httpConfig: 
+    bearerToken
+      key: password
+      name: default-webhook-secret
+    tlsConfig:
+      rootCA:
+        key: ca
+        name: default-webhook-secret
+      clientCertificate:
+        cert:
+          key: cert
+          name: default-webhook-secret
+        key:
+          key: key
+          name: default-webhook-secret
+      insecureSkipVerify: false
+
+---
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: WebhookReceiver
+metadata:
+  name: global-webhook-receiver
+  namespace: default
+  labels:
+    app: notification-manager
+    type: global
+EOF
+```
+
+#### Deploy the default DingTalkConfig and a global DingTalkReceiver
+
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+data:
+  appkey: ZGluZ2Jla3UxR2enAyeHQ=
+  appsecret: dnRFNWt2RWppOWdiZF9x
+  webhook: aHR0cHM6Ly9vYXBpLmRpbmd0YWxrLmNvbS9yb2JvdC9zZW5kP2FjY2Vzc190b2tlbj0zNjUxO
+  secret: U0VDZjJiMTkyOGUwOGY5ZjM4YzIwMmZGNiN2VhMjk1MTMyNDI0YTgxMDljMjFkYzYwNGU3MDkzNQ==
+kind: Secret
+metadata:
+  labels:
+    app: notification-manager
+  name: default-dingtalk-secret
+  namespace: default
+type: Opaque
+
+---
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: DingTalkConfig
+metadata:
+  name: default-dingtalk-config
+  namespace: default
+  labels:
+    app: notification-manager
+    type: default
+spec:
+  conversation:
+    appkey: 
+      key: appkey
+      name: default-dingtalk-secret
+    appsecret:
+      key: appsecret
+      name: default-dingtalk-secret
+    chatid: chat894f9f4d634eb283933af6c7102977b2
+  chatbot:
+    webhook:
+      key: webhook
+      name: test-user-dingtalk-secret
+    keywords: 
+    - kubesphere
+    secret:
+      key: secret
+      name: test-user-dingtalk-secret
+
+---
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: DingTalkReceiver
+metadata:
+  name: global-dingtalk-receiver
+  namespace: default
+  labels:
+    app: notification-manager
+    type: global
+EOF
+```
+
+### Customize template
+
+You can customize the message format by customize template. You need to create a template file include the template that you customized, and mount it to `NotificationManager`. Then you can change the template to the template which you defined.
```

Comment:
You can customize the message format by customizing the template. You need to create a template file include the template that you customized, and mount it to `NotificationManager`. Then you can change the template to the template which you defined.

It can set a global template, or set template for each type of receivers. If the template of the receiver does not set, it will use the global template. If the global template does not set too, it will use the default template. The default template is like this.